litellm/utils.py

```diff
@@ -5275,6 +5275,8 @@ def _map_and_modify_arg(supported_params: dict, provider: str, model: str):
             optional_params["tool_choice"] = tool_choice
         if response_format is not None:
             optional_params["response_format"] = response_format
+        if seed is not None:
+            optional_params["seed"] = seed
```

Comment:
is this correct ? I don't see it on their docs - it looks like they pass it in messages for some reason ? 


<img width="846" alt="Screenshot 2024-04-23 at 2 10 39 PM" src="https://github.com/BerriAI/litellm/assets/29436595/17093b5b-c877-4277-81ba-7c8774acbb05">

https://console.groq.com/docs/text-chat